EIPS/eip-2266.md

```diff
@@ -237,7 +237,7 @@ This proposal is fully backward compatible. Functionalities of existing standard
 
 ## Implementation
 
-Please visit [here](https://github.com/HAOYUatHZ/fair-atomic-swap/blob/master/src/atomicswap/eip2266/) to find our example implementation.
+Please visit [here](https://github.com/ethereum/EIPs/blob/master/assets/eip-2266/Example.sol) to find our example implementation.
```

Comment:
```suggestion
Please visit [here](../assets/eip-2266/Example.sol) to find our example implementation.
```
Relative path will make this link work in all environments, such as the official EIPs website.